src/mumble/UserModel.cpp

```diff
@@ -1958,18 +1958,32 @@ QString UserModel::createDisplayString(const ClientUser &user, bool isChannelLis
 	if (!user.qsFriendName.isEmpty() && user.qsName.compare(user.qsFriendName, Qt::CaseInsensitive) != 0) {
 		friendTag = QString::fromLatin1("(%2)").arg(user.qsFriendName);
 	}
+	
+	// Create a nickname-tag
+	QString nickname = user.getLocalNickname();
```

Comment:
```suggestion
	const auto nickname = user.getLocalNickname();
```

Reply:
I'm also against using `auto` in this context...